test/check-verify

```diff
@@ -201,7 +205,17 @@ def main():
     sys.stderr.write("Testing {0} for {1} ...\n".format(revision, name))
 
     try:
-        ret = step(opts, sink)
+        if opts.rebase:
+            rebase(sink)
+        if opts.install:
+            install(opts)
+
+        # Check if we are still publishing, in case some other verify
+        # machine collided with our test
+        if opts.github and sink:
+            check_publishing(sink, github)
```

Comment:
Why is check_publish called here, and not earlier?  So that the github status has time to settle down?